frontend/src/App.tsx

```diff
@@ -15,7 +15,7 @@ import { Navbar } from "./Components/Navbar";
 import { ColumnGrid } from "./Components/ColumnGrid";
 import { WorkStationGroup } from "./Data";
 
-const API_URL = "http://localhost:8000";
+const API_URL = "";
```

Comment:
Probably worth adding a comment as to why this works in prod?

Also, does this mean the frontend only works when on the same port as the webapi? That seems quite bad for development.

Reply:
This seems like it actually shouldn't be possible. Static serving and the webapi shouldn't be able to listen on the same port no matter how hard we try. I think we definitely need to be aware of this one.

Reply:
this works via caddy and reverse proxy. that’s kind of the whole point

Reply:
Ah, missed that. Now that I’ve had some time to contemplate it, I recall quite a few sites having /api/ routes that get hit on the same port as the static server (via reverse proxy) and it’s quite an elegant usage pattern. 

I’d argue here in that case that it’s nicer this way then.

Reply:
I can add a comment, but I'd prefer to keep it like this. Setting up a dev environment should be easier now we have docker compose, making it configurable gives us another thing that can break even when it works in dev, and if our api routes are on the same endpoint, I believe that means we can restrict our cors rules a lot more (at least, the only reason we have them disabled currently is because the dev server on localhost:3000 complained about communicating with localhost:8080).

We can probably make our URIs more elagant once this is by renaming `/gs-api` to `/groundstation` and `/api` to something more meaningful

Reply:
Or alternatively `/api/groundstation` and `/api/web`?